packages/ui/src/components/Icons/ErrorIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { SVGProps } from 'react';
+
+import Error from '@/assets/icons/error.svg';
+
+const ErrorIcon = (props: SVGProps<SVGSVGElement>) => {
+  return (
+    <svg
+      width="230"
+      height="230"
+      viewBox="0 0 230 230"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
+      <use href={`${Error}#error`} />
```

Comment:
https://caniuse.com/mdn-svg_elements_use_x

this is quite new, do we need a polyfill or parcel will handle it?

Reply:
Did remind me, adding  an issue LOG-2364 to eng-quality to find a better approach to handling SVG files. The current approach is not handy enough, takes too much extra config.